examples/weather_station/src/app_task.cpp

```diff
@@ -339,6 +363,121 @@ void AppTask::UpdateClusterState()
 	}
 }
 
+void AppTask::UpdatePowerSourceClusterState() {
+	EmberAfStatus status;
+	uint32_t voltage = BatteryMeasurementRead();
+	/* Value is expressed in half percent units ranging from 0 to 200. */
+	uint8_t batteryPercentage;
+	EmberAfPowerSourceStatus batteryStatus;
+	EmberAfBatChargeLevel batteryChargeLevel;
+	bool batteryPresent;
+	EmberAfBatChargeState batteryCharged;
+
+	if (voltage < 0) {
```

Comment:
This can't be true since `voltage` is unsigned.

Reply:
Changed to signed